install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -11,5 +11,7 @@ data:
       "clusterAPIControllerOpenStack": "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0",
       "clusterAPIControllerLibvirt": "docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0",
       "clusterAPIControllerBareMetal": "quay.io/openshift/origin-baremetal-machine-controllers:v4.0.0",
-      "clusterAPIControllerAzure": "quay.io/openshift/origin-azure-machine-controllers:v4.0.0"
+      "clusterAPIControllerAzure": "quay.io/openshift/origin-azure-machine-controllers:v4.0.0",
+      "baremetalOperator": "quay.io/metal3-io/baremetal-operator:latest",
```

Comment:
@russellb noticed in the last week or so that using the tag "latest" does not automatically update to the newest images on the master branch. Perhaps it does if we start tagging them with real version numbers? I think he switched to using "master" instead of "latest" to always get the latest version.

Reply:
I commented below down on the image-references file, but this can just change to the openshift image

Reply:
Done.